src/seo/MetaComponent.js

```diff
@@ -2,33 +2,34 @@ import React from "react";
 import { Helmet } from "react-helmet";
 
 function MetaComponent({ title, description, keywords }) {
-  let attentionMessage = "Come back";
-  let blinkEvent = null;
+  //THIS ARE THE CHANGES TO REMOVE THE TRANSITION EFFECT IN FAVICON.
+//   let attentionMessage = "Come back";
+//   let blinkEvent = null;
 
-  const favicon = document.getElementById("favicon");
+//   const favicon = document.getElementById("favicon");
   
-  document.addEventListener("visibilitychange", e => {
-    var isPageActive = !document.hidden;
+//   document.addEventListener("visibilitychange", e => {
+//     var isPageActive = !document.hidden;
 
-    if (!isPageActive) {
-      blink();
-    } else {
-      document.title = title;
-      clearInterval(blinkEvent);
-    }
-  });
+//     if (!isPageActive) {
+//       blink();
+//     } else {
+//       document.title = title;
```

Comment:
Do not comment the unused code. Please remove it directly

Reply:
Alright

Reply:
I did changes ,and removed extra lines of code.